Update wand protocol

CapChuyan opened the pull request:
We deploy a new set of Bribe Vaults on Story Mainnet. The new Bribe Vault is similar to projects/zoofi-io/index.js.

In this new Bribe vault, we lock asset $VIP (0x5267F7eE069CEB3D8F1c760c215569b79d0685aD), which is issued by https://www.verio.network.  The adapters convert $VIP balances to $IP using Verio's contract interfaces.

llamabutler commented:
The adapter at projects/treasury/wand.js exports TVL: 
        
 
 
    blast                     17.47 k
    sty                       137
    
    total                    17.61 k

llamabutler commented:
The adapter at projects/wand exports TVL: 
        
 
 
    blast                     1.12 M
    sty                       327.67 k
    
    total                    1.44 M

llamabutler commented:
The adapter at projects/treasury/wand.js exports TVL: 
        
 
 
    blast                     17.70 k
    sty                       125
    
    total                    17.82 k

llamabutler commented:
The adapter at projects/wand exports TVL: 
        
 
 
    blast                     1.15 M
    sty                       306.53 k
    
    total                    1.46 M

llamabutler commented:
The adapter at projects/treasury/wand.js exports TVL: 
        
 
 
    blast                     17.76 k
    sty                       123
    
    total                    17.88 k

llamabutler commented:
The adapter at projects/wand exports TVL: 
        
 
 
    blast                     1.15 M
    sty                       302.43 k
    
    total                    1.46 M

CapChuyan commented:
> We have added pricing support for vIP here [DefiLlama/defillama-server#9413](https://github.com/DefiLlama/defillama-server/pull/9413) . Please export a balance of vIP now rather than IP.

Thanks, updated.
